pkg/security/adproto/v1/activity_dump.proto

```diff
@@ -0,0 +1,151 @@
+syntax = "proto3";
+
+option go_package = "pkg/security/adproto/v1";
+
+package adproto;
+
+message ActivityDump {
+    string host = 1;
+    string service = 2;
+    string source = 3;
+
+    Metadata metadata = 4;
+
+    repeated string tags = 5;
+    repeated ProcessActivityNode tree = 6;
+}
+
+message Metadata {
+    string agent_version = 1;
+    string agent_commit = 2;
+    string kernel_version = 3;
+    string linux_distribution = 4;
+
+    string name = 5;
+    string activity_dump_version = 6;
+    bool differentiate_args = 7;
+    string comm = 8;
+    string container_id = 9;
+    uint64 start = 10;
```

Comment:
maybe we can use the built in [Timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp) type?

Reply:
Respecting the same API that is

Reply:
Alright should be done in the latest commit

Reply:
After more benchmarks, the cost is not OK for now. So we will continue using raw timestamp